linux/keyman-config/keyman_config/keyboard_details.py

```diff
@@ -323,9 +324,9 @@ def __init__(self, parent, kmp):
         prevlabel = lbl_pad
 
         # If it doesn't exist, generate QR code to share keyboard package
-        path_qr = packageDir + "/qrcode.png"
+        path_qr = path.join(tempfile.gettempdir(), kmp['packageID'] + '_qrcode.png')
```

Comment:
Would a temp path be worth adding to #5372?

Reply:
I'd put that into a separate PR if we were going to do it.

Reply:
At this point I don't see any benefits of doing that - which might change if we use it in more places :smile: